index.js

```diff
@@ -72,10 +111,24 @@ function testGenResponse(swagger, path, operation, response, config) {
 
   data.path += (swagger.basePath !== undefined ? swagger.basePath : '') + path;
 
+  // supertest url add query
+  var queryToAdd = '';
+
+  if (config.testModule === 'supertest') {
+    if (data.queryParameters.length > 0) {
+      data.path += '?';
+      data.queryParameters.forEach(function(element) {
+        queryToAdd = element.name + '=' + 'DATA' + '&';
```

Comment:
No need to split the string into multiple strings here ` queryToAdd = element.name + '=DATA&';` should be enough